DESCRIPTION

```diff
@@ -18,6 +18,9 @@ URL: https://github.com/nfidd/nfidd
 BugReports: https://github.com/nfidd/nfiidd/issues
 Depends:
     R (>= 4.2.0)
+Suggests:
+    rstan,
```

Comment:
Do we want to do this to ourselves and our course participants - i.e maybe we bite the small compilation bullet and use `cmdstanr`? The advantage of this potentially is that if they can't install `cmdstan` as a dependency they can still interact with all the material as installing deps won't fail for them?

Reply:
Ah yes that makes sense if we don't ship precompiled models (which we probably don't want).